lib/fizzy/cxx20/numeric.hpp

```diff
@@ -0,0 +1,37 @@
+// Fizzy: A fast WebAssembly interpreter
+// Copyright 2019-2020 The Fizzy Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#if __cplusplus > 201703L
+
+#include <bit>
+
+namespace fizzy
+{
+template <class T>
+constexpr int popcount(T x) noexcept
+{
+    return std::popcount.as<T>(x);
```

Comment:
What is this?

Reply:
The `using` statement didn't seem to work for functions. We still want to "copy" it from `std` to `fizzy`.

Reply:
`using` does not work because functions are constant objects, not types. This is good approach. But the `.as<T>` is wrong. And because of we cannot test it now, maybe add `static_assert()` for now?

Reply:
`static_assert` for what? The two types we support/test?

Reply:
In that case it may be better not to use a template just the two specialisations.

Reply:
I propose to this for now:
```
#if __cpp_lib_bitops
static_assert(false, "Add alias to std::popcount");
#endif 
```

As for function alias, only template function is generic function alias. There is also an option to use `constexpr`, but not for overloaded functions: https://stackoverflow.com/questions/9864125/c11-how-to-alias-a-function/49819511#49819511.

Reply:
Why don't we add a test run for C++20? There is just no point adding these things otherwise.

Reply:
None of the C++ stdlib implementations have `<bit>` support.